src/app/[locale]/(default)/about/LabOps/page.tsx

```diff
@@ -0,0 +1,124 @@
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/ui/Accordion';
+import { Card } from '@/components/ui/Card';
+import { useTranslations } from 'next-intl';
+import { unstable_setRequestLocale } from 'next-intl/server';
+import Image from 'next/image';
+
+import * as React from 'react';
+import ClientLabOpsPage from './ClientPage'; // Ensure this path is correct
```

Comment:
the clientpage component should not be in the app dir, no components should except pages and layouts etc. Also avoid default exports when possible and use named exports instead